shopsdk/README.md

```diff
@@ -0,0 +1,76 @@
+# TODO:<br>
+```
+Import the dynamic config into the code
+Write stickers script
+```
+To setup the purely configurable enviroment 6100m made, simply do this command.<br>
+```
+python3 setup.py 0
+```
+Be sure to follow all prompts.<br>
+
+There are a few services these scripts provide that you can offer on the shop!<br>
+The services it currently provides:<br>
+1. My Aquarium Mystery DLC Tanks<br>
+2. iOS 1.0 - 1.1.2 Jailbreak Server<br>
+
+To run the My Aquarium DLC script, do:<br>
+```
+python3 install.py 9 0 0 0 0 0 0 0 0
+```
+To setup the iOS 1.0 - 1.1.2 Jailbreak Server, do:<br>
+```
+bash
+clear
+clear&&python3
+```
+Then, in the python shell, do:<br>
+```
+Python 3.8.5 (default, Jul 20 2020, 19:48:14) 
+[GCC 7.5.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import utilsnotbyme as a
+>>> b=int
+>>> c=a.exploit
+>>> d=b(0)
+>>> e=b(1)
+>>> c(d,e)
+```
+Already have a hex.dat file? Do this instead:<br>
+```
+Python 3.8.5 (default, Jul 20 2020, 19:48:14) 
+[GCC 7.5.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import utilsnotbyme as a
+>>> b=int
+>>> c=a.exploit
+>>> d=b(0)
+>>> e=b(2)
+>>> c(d,e)
+```
+Want to download a hex.dat template? Do this instead:<br>
+```
+Python 3.8.5 (default, Jul 20 2020, 19:48:14) 
+[GCC 7.5.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import utilsnotbyme as a
+>>> b=int
+>>> c=a.exploit
+>>> d=b(0)
+>>> e=b(3)
+>>> c(d,e)
+```
+CREDITS.txt missing? Please do:<br>
+```
+Python 3.8.5 (default, Jul 20 2020, 19:48:14) 
+[GCC 7.5.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import utilsnotbyme as a
+>>> b=int
+>>> c=a.exploit
+>>> d=b(0)
+>>> e=b(4)
+>>> c(d,e)
+```
+-WL24<br>
+Crushing the WC24 service competition!<br>
```

Comment:
no u

Reply:
Heh, that was a joke, I'll get rid of that though.